packages/compass-collection/package.json

```diff
@@ -59,6 +59,7 @@
     "@mongodb-js/compass-app-stores": "^7.7.4",
     "@mongodb-js/compass-components": "^1.20.2",
     "@mongodb-js/compass-logging": "^1.2.8",
+    "@mongodb-js/compass-workspaces": "^0.2.4",
```

Comment:
Whoops!

```
lerna WARN ECYCLE Dependency cycles detected, you should fix these!
lerna WARN ECYCLE @mongodb-js/compass-collection -> @mongodb-js/compass-workspaces -> @mongodb-js/compass-collection
lerna WARN ECYCLE @mongodb-js/compass-workspaces -> @mongodb-js/compass-databases-collections -> @mongodb-js/compass-workspaces
```

Reply:
Had to change the type dependency and move workspace types to workspaces plugin (https://github.com/mongodb-js/compass/pull/5213/commits/5320220b8451532e7b444fb14ccf0063b0d119d0) to make sure we're not running into a circular dependency issue